locales/lv/messages.json

```diff
@@ -24,6 +24,7 @@
     },
     "storageDeviceNotReady": {
         "message": "Atmiņas ierīce nav gatava. MikroSD kartes gadījumā, pārliecināties, ka lidojuma kontrolieris ir atbilstoši atpazinis to."
+        "description": "Edited in crowdin by EGParadox."
```

Comment:
This needs a comma at the end of the previous line or it will break the JSON format.